core/src/com/mygdx/minigolf/Game.java

```diff
@@ -1,33 +1,135 @@
+/*
+ *	Changes in this file is for testing purposes only, and will be removed
+ */
+
 package com.mygdx.minigolf;
```

Comment:
Endinger i denne filen er for å teste InputHandler, og vil bli fjernet (hvis vi ikke vil bruke noe av det da).